deps/hiredis/examples/example.c

```diff
@@ -57,7 +57,7 @@ int main(int argc, char **argv) {
     for (j = 0; j < 10; j++) {
         char buf[64];
 
-        snprintf(buf,64,"%d",j);
+        snprintf(buf,64,"%u",j);
```

Comment:
Already fixed in https://github.com/redis/hiredis/commit/f932f0ed275bf9d6653b11694e39a13f57e7dc54